manager/engine.go

```diff
@@ -161,17 +162,35 @@ func (m *VolumeManager) BackupSnapshot(backupName, volumeName, snapshotName stri
 		return err
 	}
 
+	// Create the BackupVolume CR if not present
+	backupVolume, err := m.ds.GetBackupVolumeRO(volumeName)
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+		bv := &longhorn.BackupVolume{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: volumeName,
+			},
+		}
+		backupVolume, err = m.ds.CreateBackupVolume(bv)
+		if err != nil && !apierrors.IsAlreadyExists(err) {
+			return err
+		}
+	}
```

Comment:
There might be a case that if the BackupVolume created at here, and the backup volume config haven't be created at the remote backup target.
At the moment, the backup_target_controller think this BackupVolume CR must be deleted since it's only in the remote backup target but only within in-cluster.
If the cascading deletion also implemented, the corresponding Backup CRs will be deleted (also the remote backup config)

Reply:
Since you decide to remove the cascading deletion, the BackupVolume CR creation always happens in the controllers. You can ignore this comment.